config/config.go

```diff
@@ -422,6 +441,17 @@ func CheckConfig() {
 		netclient.OS = runtime.GOOS
 		saveRequired = true
 	}
+	slog.Info("OS is", "os", netclient.OS)
+	if netclient.OS == "linux" {
+		initType := GetInitType()
+		slog.Debug("init type is", "type", initType.String(), "old type", netclient.InitType.String())
```

Comment:
yes it can be, but that would be an unusual situation.  Normally the init system is setup on initial install and never changed.   some distro (artix) allow you to change the init system.